code/modules/mob/living/simple_animal/hostile/wendigo.dm

```diff
@@ -0,0 +1,236 @@
+/*	TODO:
+	Let them evolve and heal from eating. Should a carbon they eat go over a certain damage threshold and
+	the wendigo is at full health. Strip and delete the corpse.
+	Human wendigo->1 eat->Wendigo->3 eat->Alpha Wendigo (Can only be ONE at any time)
+	They don't like fire. Will stay away from fire (As far as their light radius -1) and won't target anyone
+	in vicinity of said fire
+	Can mimic voices, steal poly code, and change it (Maybe just keep a log of people consumed,
+	and use their name plus shout for help)
+	Butcherable for one piece of meat only. When consumed (Don't use reagents) will begin to turn the person
+*/
+#define HUMEVOLV 1
+#define EVOLEVOLV 3
+
+
+/mob/living/simple_animal/hostile/wendigo
+	name = "wendigo"
+	desc = "Thought to be just a cautionary tale, now moreso than ever."
+	icon_state = "wendigo"
+	icon_living = "wendigo"
+	icon_dead = "wendigo_dead"
+
+	health = 150
+	maxHealth = 150
+	speed = 1
+	stat_attack = DEAD //Gotta chow down somehow
+	vision_range = 12 //Slightly larger vision range
+	harm_intent_damage = 8
+	melee_damage_lower = 15
+	melee_damage_upper = 20
+	attacktext = "bites"
+	attack_sound = 'sound/weapons/bite.ogg'
+	meat_amount = 1 //Only one piece of meat
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/wendigo
+	var/list/names = list() //List of names of the people it's eaten
+	var/consumes = 0 //How many people it has eaten
+	environment_smash = 1
+	speak = list("Help!","Help me!","Somebody help!","Get over here, quickly!")
```

Comment:
I'm not sure how well this is supposed to work.  For someone to hear it they have to see it on screen and will probably notice a windego.